styles/_govuk-elements.scss

```diff
@@ -0,0 +1,30 @@
+// GOV.UK elements
+@import "govuk-elements-sass/public/sass/elements/helpers";                       // Helper functions and classes
```

Comment:
Unfortunately, I couldn't just import everything because of the way govuk-elements is designed.  Also I had to be explicit in the paths